aiozipkin/helpers.py

```diff
@@ -37,3 +44,41 @@ def create_endpoint(servce_name, *, ipv4=None, ipv6=None, port=None):
 def make_timestamp(ts=None):
     ts = ts if ts is not None else time.time()
     return int(ts * 1000 * 1000)  # microseconds
+
+
+def make_headers(context):
+    headers = {
+        TRACE_ID_HEADER: context.trace_id,
+        SPAN_ID_HEADER: context.span_id,
+        PARENT_ID_HEADER: context.parent_id,
+        FLAGS_HEADER: '0',
+        SAMPLED_ID_HEADER: '1' if context.sampled else '0',
+    }
+    return headers
+
+
+_required_headers = (TRACE_ID_HEADER, PARENT_ID_HEADER, SPAN_ID_HEADER)
+
+
+def _parse_sampled(headers):
+    sampled = headers.get(SAMPLED_ID_HEADER, None)
+    if sampled is None or sampled == "":
```

Comment:
`if not sampled`?  It seems like header values are strings, so `bool(sampled)` below going to be true all the time. Is that correct?

Reply:
good catch!

Reply:
Added TODO for more test cases, for now I am trying to play with API and figure out aiohttp integration;